content/en/docs/Config/webhook.md

```diff
@@ -50,4 +51,46 @@ Then, in Argus' `config.yml`
           CUSTOM_HEADER_NAME: hello123
 ```
 
-would execute ["SOMETHING.SH" "hello123"]
\ No newline at end of file
+would execute ["SOMETHING.SH" "hello123"]
+
+## Using GitLab Webhooks
+GitLab webhooks can be used to start GitLab CI pipelines through which the applications are updated.
+For this, a `Pipeline trigger` must be created in GitLab in the project settings under `CI/CD`.
+
+[Offical Docs](https://docs.gitlab.com/ee/ci/triggers/#use-a-webhook)
+
+The `token` must be stored as secret and doesn't have to be added to the URL. 
+
+Example URL: `https://gitlab.com/api/v4/projects/<project_id>/ref/<ref_name>/trigger/pipeline`
+
+config.yaml
```

Comment:
```suggestion
config.yml
```
Not major and I know it is yaml, but I'm too used to yml and that's how it's written everywhere else on the site

Reply:
Done